File: Source/UnitTests/TestCode/AccessorRefactoringTest/InterfaceSetterGetter.java
package Test;
public interface IShape
{
	int getLength();
	void setLength(int length);
}

public abstract class Shape implements IShape
{
	public abstract int getLength();
	public abstract void setLength(int length);
}

public class Rectangle extends Shape
{
	private int length;
	public int getLength()
	{
		return length;
	}
	public void setLength(int length)
	{
		this.length = length;
	}
}

File: Source/UnitTests/TestCode/AccessorRefactoringTest/InterfaceSetterGetter.cs
namespace Test
{
	public interface IShape
	{
		int Length { get; set; }
	}

	public abstract class Shape : IShape
	{
		public abstract int Length{ get; set;}
	}

	public class Rectangle : Shape
	{
		private int length;
		public int Length
		{
			get { return length; }
			set { this.length = value;}
		}
	}
}